.github/workflows/check-e2e.yml

```diff
@@ -36,10 +36,11 @@ jobs:
         if: steps.playwright-dep-cache.outputs.cache-hit != 'true'
         run: npx playwright install chromium firefox
 
-      - name: Start http-api test server
+      - name: Install rad binaries
         run: |
           mkdir -p tests/artifacts;
-          ./scripts/run-httpd-with-fixtures --non-interactive --download 2>&1 | tee tests/artifacts/httpd-${{ matrix.browser }}.log &
+          ./scripts/install-binaries;
+          ./scripts/get-binaries-folder >> $GITHUB_PATH;
```

Comment:
Could we maybe move this to the other script and hide the functionality behind a flag? I feel like both scripts have an implicit contract about how to construct the path, so it would be nice to have it all in one place.

Something like: `install-binaries --show-path`.

Reply:
One other reason I added this is so we get all the download logs, etc from the `install-binaries` script and then just need the `get-binaries-folder` one for the CI output..

So if we add a `--show-path` flag, this would need to be something like `--show-only-path` so we can pipe that path into `$GITHUB_PATH`

If we want to have the path construction in one place, maybe it would be better to make `install-binaries` query `get-binaries-folder` for the correct path instead of computing it by itself?

Reply:
Can't we run it twice? Like:

```
./scripts/install-binaries
./scripts/install-binaries --show-path
```

Where the second invocation only prints the path, but doesn't download anything.

Reply:
ah yeah that would work 👍

Reply:
fixed in 3c9fbd7
Also added a small help